okhttp-testing-support/src/main/kotlin/okhttp3/RecordingEventListener.kt

```diff
@@ -129,7 +129,7 @@ open class RecordingEventListener : EventListener() {
     for (lock in forbiddenLocks) {
       assertThat(Thread.holdsLock(lock))
           .overridingErrorMessage(lock.toString())
-          .isFalse()
+          .isFalse
```

Comment:
The IntelliJ waning irks me here, and I prefer the quotes. The is false() call has a critical side-effect, but accessing a property should not have side-effects.

I'd prefer to leave these as functions.